test/application/respond.js

```diff
@@ -20,7 +20,7 @@ describe('app.respond', () => {
         res.statusCode = 200;
         setImmediate(() => {
           res.setHeader('Content-Type', 'text/plain');
-          res.end('lol');
+          res.end('Hello');
```

Comment:
The better way to fix this is add `res.setHeader('Content-Length', '3')` here to make it more clearly

Reply:
sure, but I find that more confusing since you set `body = 'Hello'`

Reply:
not quite sure what the purpose of `body = ` is? Maybe thats' what should be removed?

Reply:
If we let koa handle the respond, then `ctx.body=` will both set content-length and body.

I think the test case want to test the situation that set respond manually can override koa's default behavior.

Reply:
Fixed